contact.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="styles.css" />
+  <title>CONTACT | A K BROTHERS</title>
+</head>
+
+<body>
+  <!-- Hero section -->
+  <header class="contact-hero">
+    <div class="hero-content contact-container">
+      <div class="contact-heading hero-text-main">
+          Have a question for us?
+          Go for it.
+      </div>
+      </div>
+     </header>
+
+<!-- Section: Contact -->
+<section id="contact">
+     <div class="contact-form bg-primary p-2">
+       <h2 class="m-heading">Get in touch.</h2>
+         <div class="form-group">
+           <label for="name">Name</label>
+           <input type="text" name="name" id="name" placeholder="Enter your name">
+         </div>
+         <div class="form-group">
+           <label for="phone">Phone Number</label>
+           <input type="text" name="phone" id="phone" placeholder="Enter your contact number for callback">
+         </div>
+         <div class="form-group">
+           <label for="email">Email</label>
+           <input type="email" name="email" id="email" placeholder="Enter your email">
+         </div>
+         <div class="form-group">
+           <label for="message">How can we help you?</label>
+           <textarea name="message" id="message" placeholder="I was wondering about availability and rates. I need help with the following:"></textarea>
+         </div>
+         <input type="submit" value="Send" class="btn btn-dark">
```

Comment:
- change it to button
- make the text similar to the one in the reference site. (capitalise it)
- add transition